src/lib.rs

```diff
@@ -30,7 +29,7 @@ pub use term::load_book::load_file_to_book;
 
 pub const ENTRY_POINT: &str = "main";
 pub const HVM1_ENTRY_POINT: &str = "Main";
-pub const CORE_BUILTINS: [&'static str; 1] = ["HVM.log"];
+pub const CORE_BUILTINS: [&'static str; 2] = ["HVM.log", "HVM.black_box"];
```

Comment:
Shouldn't this be coming from the hvm-core crate?

Reply:
The name is a bit misleading. These two functions are not implemented in `hvm-core`. They are implemented by `hvm-lang`, in `src/lib.rs:38`, by the `create_host` function which creates a `hvmc::host::Host` and adds the builtin defs to it.

What characterizes them is that they are definitions that are not in the `hvml` `Book`, but at the same time, they are in the `hvmc` `Book`. That's why I named the constant `CORE_BUILTINS`. Perhaps a better name would be `VM_BUILTINS`?

Reply:
Sorry, I should've paid more attention. I think the name is fine, you can just add a comment describing this that you just wrote.